classes/output/page_managedocs_list.php

```diff
@@ -238,11 +238,10 @@ protected function export_version_for_template($output, $policy, $version, $stat
         }
         if ($status == policy_version::STATUS_ARCHIVED) {
```

Comment:
Can't we simply remove the condition on the Edit action block above (around line 202)?

Reply:
the labels are still different

Reply:
Oh yes, I see. Thanks.